src/main/java/reposense/model/AuthorConfiguration.java

```diff
@@ -329,11 +329,11 @@ public boolean containsName(String name) {
         return authorNamesToAuthorMap.containsKey(name.toLowerCase()) || authorEmailsToAuthorMap.containsKey(name);
     }
 
-    public static void setHasAuthorConfigFile(boolean hasAuthorConfigFile) {
-        AuthorConfiguration.hasAuthorConfigFile = hasAuthorConfigFile;
+    public void setHasAuthorConfigFile(boolean hasAuthorConfigFile) {
+        this.hasAuthorConfigFile = hasAuthorConfigFile;
     }
 
-    public static boolean hasAuthorConfigFile() {
-        return hasAuthorConfigFile;
+    public boolean hasAuthorConfigFile() {
+        return this.hasAuthorConfigFile;
```

Comment:
Minor point but iirc we don't use 'this' if the reference is not ambiguous, so just `return hasAuthorConfigFile`.

Reply:
Okay, I've updated it, thanks for noticing that!